CMakeLists.txt

```diff
@@ -128,43 +128,21 @@ endif()
 ###############################################################################
 ### FMS
 ###############################################################################
-include(fms_files.cmake)
-add_library(fms ${fms_src_files} ${fms_headers})
-# stupid cmake can not figure out dependency of fft.F90 on fft99.F90 because 'use fft99_mod' is inside ifdefs
-set_property(SOURCE FMS/fft/fft.F90 APPEND_STRING PROPERTY COMPILE_FLAGS "-DSGICRAY=0 -DNAGFFT=0")
-
-list(APPEND _fms_defs_public use_libMPI
-                             use_netCDF
-                             GFS_PHYS
-                             INTERNAL_FILE_NML)
+set(GFS_PHYS ON CACHE BOOL "Enable GFS Physics")
+if(NOT 32BIT)
+   set(64BIT ON CACHE BOOL "Enable 64-bit")
+endif()
 if(QUAD_PRECISION)
-  list(APPEND _fms_defs_public ENABLE_QUAD_PRECISION)
+  set(ENABLE_QUAD_PRECISION ON CACHE BOOL "Enable Quad-precision")
```

Comment:
The default for this is also ON.
So, if `QUAD_PRECISION` is not ON, we need to change this.  In other words, we need to change this to:
```
if (NOT QUAD_PRECISION)
    set(ENABLE_QUAD_PRECISION OFF CACHE BOOL "Enable Quad-precision")
endif()
```

Reply:
@aerorahul, good point.  I made the changes to CMakeLists.txt.  Unfortunately the cpld_2threads test still fails